### ISSUE TITLE
[Issue]: Cannot read generation parameters when the webUI has just been launched (most of the time)

### ISSUE THREAD

**Issue by StableInfo** [vladmandic/automatic#1421]
### Issue Description

If the UI has been just launched, then pasting some text generation parameters and pushing the arrow button, the UI will be condemned to "load" indefinitely, even restarting the tab will not save it. You need to restart from the .bat file all over.

Only way to skip this problem, is to write a line or 2 in the positive and negative prompt areas and generate random images to start with, once or twice is enough, now you can read generation parameters with no problem for the rest of he time vlad auto is launched.
I tried to search and did not find anything equal to this issue. Hope this helps.

### Version Platform Description

Firefox.

### Acknowledgements

- [X] I have read the above and searched for existing issues

**Comment by vladmandic** [comment]
i cannot reproduce.
are you sure your workaround steps are necessary? might be that ui is simply not fully initialized yet.
and never ever ever omit version info. i have to mark issue invalid.

**Comment by vladmandic** [comment]
version is a) printed in console log on startup, b) available in ui in system info tab

**Comment by StableInfo** [comment]
Version I think:
![image](https://github.com/vladmandic/automatic/assets/136593585/c2469c82-5a95-4986-a920-0a9795d6d389)

**Comment by vladmandic** [comment]
that is ancient.

**Comment by StableInfo** [comment]
So using the "setup" file in the vlad auto rep was enought to make it updated? I am surprised
I suppose I need to add a "git pull" somewhere right?

**Comment by vladmandic** [comment]
No, there is `--upgrade`flag to upgrade, you don't need to run any manual commands.

**Comment by StableInfo** [comment]
Hello vlad, sorry, I dont understand, what should I do exaclty to upgrade my vlad ui?

**Comment by vladmandic** [comment]
check FAQ, this is one of the most trivial and common questions: #1011

**Comment by StableInfo** [comment]
Oh ok, so its like the old parameters on the original ui, except it has NOT been moved to the settings,  and you dont write it inside inside the bat file like the old ui, so apparently I have to run this here:
.. \vlad\automatic>webui.bat --upgrade

Except I got an error oh well:

launch.py: error: unrecognized arguments: --upgrade

**Comment by vladmandic** [comment]
ah, because your installation is soo old that it doesn't even recognize command line flags added long time ago.
try doing manual update once using `git pull` in your installation folder and then follow-up with `webui --upgrade`

**Comment by vladmandic** [comment]
we can do step-by-step to get you up to date and from there you can use `--upgrade` moving forward,
but easiest for everyone is if you just backup your `models` and `outputs` folders so you don't loose them and then do a full reinstall.
